doc/source/user_guide/codelist.rst

```diff
@@ -0,0 +1,209 @@
+.. _codelist:
+
+Codelist
+========
+
+A "codelist" (named in reference to code lists in `SDMX <https://sdmx.org/>`_) is a yaml
+file that contains a list of allowed values for any **index** dimension of the IAMC
+format (model, scenario, region, variable). Which index dimension a code list will
+be applied to is determined by the **name of the folder** in which it is located. For
+example, if a codelist is located in a folder called ``model/`` it will be applied to
+the "model" index dimension. 
+
+A codelist must only contain a single list of allowed values. In the most simple case
```

Comment:
What is the message here?
"A codelist must only contain a single list of allowed values."

Reply:
The message was supposed to be that a codelist can only be a single list meaning something like this:

```yaml
allowed_regions:
  - region_a
  - region_b
  - ...
allowed_variables:
  - variable_a
  - ...
```

would not be allowed as this would be two lists (which are part of a mapping).
But you're right, the sentence probably does more harm than good and I think it can be removed without loss of information.